v20.2/alter-role.md

```diff
@@ -32,14 +32,79 @@ table td:first-child {
+`CANCELQUERY`/`NOCANCELQUERY` | Allow or disallow the role to cancel queries and sessions of other roles. Without this privilege, roles can only cancel their own queries and sessions. Even with this privilege, non-admins cannot cancel admin queries or sessions. This option should usually be combined with `VIEWACTIVITY` so that the role can view other roles' query and session information. <br><br>By default, the parameter is set to `NOCANCELQUERY` for all non-admin roles.
+`VIEWACTIVITY`/`NOVIEWACTIVITY` | Allow or disallow a role to see other roles' queries and sessions using `SHOW QUERIES`, `SHOW SESSIONS`, and the Statements and Transactions pages in the Admin UI. Without this privilege, the `SHOW` commands only show the role's own data and the Admin UI pages are unavailable. <br><br>By default, the parameter is set to `NOVIEWACTIVITY` for all non-admin roles.
+`CONTROLCHANGEFEED`/`NOCONTROLCHANGEFEED` | Allow or disallow the role to run `CREATE CHANGEFEED` on tables they have `SELECT` privileges on. <br><br>By default, the parameter is set to `NOCONTROLCHANGEFEED` for all non-admin roles.
+`MODIFYCLUSTERSETTING`/`NOMODIFYCLUSTERSETTING` | Allow or disallow the role to to modify the cluster settings with the `sql.defaults` prefix. <br><br>By default, the parameter is set to `NOMODIFYCLUSTERSETTING` for all non-admin roles.
 
 ## Examples
 
+{{site.data.alerts.callout_info}}
+The following statements are run by the `root` user that is a member of the `admin` role and has `ALL` privileges.
+{{site.data.alerts.end}}
+
+### Allow a role to log in to the database using a password
+
+~~~ sql
+root@:26257/defaultdb> ALTER ROLE carl WITH LOGIN PASSWORD 'An0ther$tr0nGpassW0rD' VALID UNTIL '2021-10-10';
+~~~
+
+### Prevent a role from using password authentication
+
+The following statement prevents the user from using password authentication and mandates certificate-based client authentication:
+
+{% include copy-clipboard.html %}
+~~~ sql
+root@:26257/defaultdb> ALTER ROLE carl WITH PASSWORD NULL;
+~~~
+
+### Allow a role to create other roles and manage authentication methods for the new roles
+
+~~~ sql
+root@:26257/defaultdb> ALTER ROLE carl WITH CREATEROLE CREATELOGIN;
+~~~
+
+### Allow a role to create and rename databases
+
+~~~ sql
+root@:26257/defaultdb> ALTER ROLE carl WITH CREATEDB;
+~~~
+
+### Allow a role to pause, resume, and cancel non-admin jobs
+
+~~~ sql
+root@:26257/defaultdb> ALTER ROLE carl WITH CONTROLJOB;
+~~~
+
+### Allow a role to see and cancel non-admin queries and sessions
+
+~~~ sql
+root@:26257/defaultdb> ALTER ROLE carl WITH CANCELQUERY VIEWACTIVITY;
+~~~
+
+### Allow a role to control changefeeds
+
+~~~ sql
+root@:26257/defaultdb> ALTER ROLE carl WITH CONTROLCHANGEFEED;
+~~~
+
+### Allow a role to modify cluster settings
+
+~~~ sql
+root@:26257/defaultdb> ALTER ROLE carl WITH MODIFYCLUSTERSETTING;
+~~~
+
+
+<!--
```

Comment:
Suggest deleting this commented-out section.  Git will have it if we ever need it.